src/Orchard.Web/Core/Common/Views/CommonMetadataLastModified.cshtml

```diff
@@ -7,7 +7,15 @@
 @using Orchard.Core.Common.Models;
 @{
     CommonPart commonPart = Model.ContentPart;
-    string lastModifiedBy = commonPart.As<CommonPart>() == null ? null : commonPart.As<CommonPart>().VersionModifiedBy;
+    string displayUser = null;
+    if (commonPart.As<CommonPart>() != null) {
```

Comment:
I was following the previous code which had the cast. Removed and seems to be fine. Updating